website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -117,6 +117,16 @@ Don't use alerts to communicate error messages that are not caused by the user.
 </Hds::Alert>
 !!!
 
+### Informational Alerts
+
+Use Informational alerts, like highlight and neutral variants, for less urgent messages. These include updates, promotions, and non-urgent warnings. These alerts are non-disruptive. They suit info messages or up-sells that don't need immediate attention. Use critical, success, or warning alerts instead for errors or urgent situations.
+
+!!! Info 
+
+Since there isn't a Figma counterpart for these, it’s important to collaborate with your engineering partner to ensure proper use and accessibility.
```

Comment:
[question] As a reader/consumer, I am not sure what I am supposed to do here. I know/see that there are `highlight` and `neutral` variants in Figma, so what am I supposed to do? what is missing? what should I do/discuss with my engineering partner? I would be confused 🤔

Reply:
That is true. I'm going to add some more details about this.

Reply:
The defaults have been updated in the associated PR: https://github.com/hashicorp/design-system/pull/2500

So for that reason, there is unlikely anything that engineering partners would need to do. I would just explain about the default behaviors, i.e., that the neutral & variants do not include `role="alert"` and `aria-live="polite"` as the other Alert colors do so that they will not unnecessarily disrupt users.

You could perhaps leverage some of @MelSumner's language in the Jira ticket as well:
"Since an element with the role of alert or alertdialog (i.e., our alert component) is read to the user before any other content is read, alerts should specifically be saved for important things such as errors, server issues, etc."